packages/utilities/src/focus.ts

```diff
@@ -100,7 +100,7 @@ export function getPreviousElement(
       tabbable);
 
     if (childMatch) {
-      if ((tabbable && (isElementTabbable(childMatch, true))) || !tabbable) {
+      if ((tabbable && isElementTabbable(childMatch, true)) || !tabbable) {
```

Comment:
Can you add a comment or change the name tabbable. This was named confusingly and if you are up to changing it that would be great. Tabbable makes me think that the element is tabbable, rather than something like elementNeedsToBeTabbable, which is what it really corresponds to. If you'd rather just add a comment to make that more clear that's cool too.